pages/posts/2.tsx

```diff
@@ -0,0 +1,179 @@
+  }, [])
+
+  return (
+    <>
+      <Head>
+        <title>Wesh Network</title>
+        <meta name="description" content="Wesh Network" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <main className={styles.Body}>
+        <Navbar />
+
+        <section className={styles.SectionBlog}>
+          <div className={styles.SectionBlogContainer}>
+            <div className={styles.SectionBlogCard}>
+              <div className={styles.SectionBlogText}>
+                <h2>{posts.title}</h2>
+                <p>
+                  Now that you have been introduced to Wesh in the
+                  <a href="https://wesh.network/posts/1"> previous blog post</a>, it’s time to explore the code. We will write a “Hello World” app, similar to the
+                  <a href="https://go.dev/doc/tutorial/getting-started"> Go tutorial </a>
+                  which you can read to set up Go on your computer and for other details. (This has been tested with Go 1.19 on macOS and Ubuntu.) This example app will connect to
+                  the Wesh service and call a function. When finished, you will be confident that you can use Wesh on your platform.
+                </p>
+                <p>Similar to the Go tutorial, in a terminal enter:</p>
+                <pre>
+                  <code className="language-js">
+                    {`
+$ cd mkdir hello
+$ cd hello
+$ go mod init example/hello
+`}
+                  </code>
+                </pre>
+                <p>In your text editor, create a file hello.go in which to write your code. Paste the following code into your hello.go file.</p>
+                <pre>
+                  <code className="language-js">
+                    {`package main
+
+import (
+	"context"
+	"fmt"
+
+	"berty.tech/weshnet"
+	"berty.tech/weshnet/pkg/protocoltypes"
+)`}
+                  </code>
+                </pre>
+                <p>
+                  We import <code className="language-js">&quot;mft&quot;</code> as usual in Go code so that we can print a message. We import
+                  <a href="http://berty.tech/weshnet"> berty.tech/weshnet </a> so that we can call the Go function to initialize the Wesh service (explained below). Finally, your
+                  app interacts with the Wesh service through gRPC which uses Protobuf messages. Therefore, we import the Wesh API Protobuf types with
+                  <a href="http://berty.tech/weshnet/pkg/protocoltypes"> berty.tech/weshnet/pkg/protocoltypes </a>, and we import
+                  <code className="language-js">&quot;context&quot;</code> because all of the gRPC calls use a Go context object (explained below).
+                </p>
+                <p>To complete the example, paste the following main function and save the file.</p>
+                <pre>
+                  <code className="language-js">
```

Comment:
I'm curious. Does there already exist a "language-go"? If yes, then can we use it? If not, then don't worry about it.

Reply:
OK. Thanks for checking.